src/ui/components/diary/EmptyInputField.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from '@emotion/styled';
+import { styleToken } from '@ui/styles';
+import { PropsWithChildren } from 'react';
+
+type EmptyInputFieldProps = {
+  id: string;
+  onClick: () => void;
+};
+
+export function EmptyInputField({ id, onClick, children }: EmptyInputFieldProps & PropsWithChildren) {
```

Comment:
P3:

```suggestion
type EmptyInputFieldProps = {
  id: string;
  onClick: () => void;
} & PropsWithChildren;

export function EmptyInputField({ id, onClick, children }: EmptyInputFieldProps) {
```

다른곳에도 Component Props를 한번에 받고있는것으로 보여져요.
`EmptyInputFieldProps` 같은 코드가 다른곳에도 사용되는것이 아니라면 `type` 에 적어주시면 좋겠네요.